doc/install_windows.txt

```diff
@@ -36,61 +36,13 @@ generated by Theano).
-
-Unfortunately Microsoft recently stopped distributing Visual Studio
-Express 2008 (the compilers required for Python 2.7 are provided,
-though), therefore we require a temporary install of Visual Studio Express
-2010 to be able to install CUDA (its installer requires a Visual
-Studio installation to proceed). Afterwards, the Visual Studio 2010
-can be safely removed. If someone knows how to install CUDA 5.5
-without a proper Visual Studio installation, please let us know.
-
-First we need to install Microsoft Visual Studio 2010 Express, which
-is required to install CUDA. You can download it from
-`Visual Studio Express
-<http://www.visualstudio.com/en-us/products/visual-studio-express-vs.aspx>`_.
-Please install the Visual C version. We have downloaded the
-`all-in-one CD <http://go.microsoft.com/?linkid=9709969>`_, extracted
-it using `7zip <http://www.7-zip.org/>`_, and ran the installer at
-VCExpress\\setup.exe.
-
-If you want a 64bit Python installation, Visual Studio 2010 Express
-doesn't provide a 64bit compiler. To get one download and install the
-`Windows Software Development Kit version 7.1
-<http://msdn.microsoft.com/en-us/windowsserver/bb980924.aspx>`_.
-
-Now you have a running (and free even for commercial use) installation
-of MSVS2010 IDE with 32- and 64-bit compilers.
-
-Once Visual Studio is installed, you can install CUDA. We recommend
-CUDA 5.5, as it supports MSVC 2008. Download the CUDA installer from
-`the CUDA archive
-<https://developer.nvidia.com/cuda-toolkit-55-archive>`_. Be sure to
-get 32-, or 64-bit version depending on your system configuration.
-
-Once CUDA is installed you can remove VisualStudio Express 2010.
-
-Finally, grab the `Microsoft Visual C++ Compiler for Python 2.7
-<http://www.microsoft.com/en-us/download/details.aspx?id=44266>`_. It
-provides the now-obsolete compilers form Visual Studio 2008 that are
-required for compatibility with Python 2.7. To install the package for
-all users please:
-
-1. open an administrator's console (got to ``start``, then type ``cmd``,
-   right-click on the command prompt icon and select ``run as
-   administrator``)
-2. ``cd`` to your downloads directory and execute ``msiexec /i
-   VCForPython27.msi ALLUSERS=1``
-
-The package will be installed to ``C:\Program Files
-(x86)\Common Files\Microsoft\Visual C++ for Python\9.0``.
-
-Finally download the ``stdint.h`` header from
-`here <https://sourceforge.net/p/mspgcc/msp430-libc/ci/master/tree/include/stdint.h>`_ and save it as
-``C:\Program Files (x86)\Common Files\Microsoft\Visual C++ for
-Python\9.0\VC\include\stdint.h``.
+libgpuarray and PyGPU
+#####################
+
+Theano uses libgpuarray to interact with graphic cards (such as NVIDIA cards) at low level,
+and PyGPU as a Python wrapper around libgpuarray. Instructions to compile and install
+both packages are available at the `GpuArray installation page
+<http://deeplearning.net/software/libgpuarray/installation.html#windows-specific-instructions>`_.
```

Comment:
Maybe we want to mention anaconda here, since it makes things much easier for people who want to install those.  Alternatively, I could mention the anaconda packages in the libgpuarray docs.